plaso/engine/worker.py

```diff
@@ -218,22 +222,21 @@ def _AnalyzeFileObject(self, mediator, file_object):
 
       data = file_object.read(maximum_read_size)
 
-    display_name = mediator.GetDisplayName()
+    analyzer_attributes = {}
```

Comment:
Remove this and directly set attributes on event_data_stream instead.

Reply:
I have this as an after care point of https://github.com/log2timeline/plaso/issues/2956. I opt to do this in a different PR, to not change everything in a single PR.

Reply:
I think your aftercare point is a bit different, I meant changing what's currently on like 241 to something like `setattr(event_data_stream, result.attribute_name, result.attribute_value)`. The aftercare point that I could identify from #2956 that I think you're talking about is `To do - determine if analyzers should set EventDataStream attributes directly?` which is a more involved change.
I'm just asking to remove the extra loop in this method.

Reply:
ah, yeah that is different than what I interpreted, I'll change it

Reply:
done